src/hooks/events.ts

```diff
@@ -1,12 +1,16 @@
 import { useAsync } from "react-use";
 
-import { fetchAllVenueEvents } from "api/events";
+//import { fetchAllVenueEvents } from "api/events";
 
 import { ReactHook } from "types/utility";
 import { VenueEvent } from "types/venues";
 
 import { WithId, WithVenueId } from "utils/id";
-import { tracePromise } from "utils/performance";
+
+//import { tracePromise } from "utils/performance";
```

Comment:
Commenting these out directly isn't making use of `SCHEDULE_LOADFROM_GS`?

Reply:
Code which used them is commented out and replaced with use of cache below

Reply:
@yarikoptic I understand that, I can read code :) Yet this PR explicitly states that `SCHEDULE_LOADFROM_GS` was added so it could be used to conditionally use the old method or the new. Which isn't the case here since it's commented out. Thus why I called it out.

Reply:
You can't conditionally render a hook. But you can do things within the hook conditionally.

Reply:
I tried to make the dependency array conditional (and it worked) but then the linter didn't let me commit it. I don't know if it breaks something with react or not.. but the way it is written now works and it makes loading of the schedule much quicker.